app/utils/calculation/calculation.ts

```diff
@@ -136,11 +136,34 @@ const getPeriodicityText = ({
   return periodicityText;
 };
 
+enum MeasureType { mass = "mass", length = "length", }
```

Comment:
Should be moved in `app/types` folder (just create something like guides.ts and import it here)

Reply:
Made app/types/measureType.ts in the most recent push